docs/app-framework-development-guide.md

```diff
@@ -91,13 +91,13 @@ virtual void offerRescinded(SchedulerDriver* driver, const OfferID& offerId);
  * Invoked when the status of a task has changed (e.g., a slave is
  * lost and so the task is lost, a task finishes and an executor
  * sends a status update saying so, etc). If implicit
- * acknowledgements are being used, then returning from this
+ * acknowledgments are being used, then returning from this
```

Comment:
I'm inclined to leave in the extra "e" since I believe both spellings are acceptable FWICT and we have more with the extra "e":

➜  mesos git:(master) ✗ grep -Ri acknowledgements src | wc -l
     149
➜  mesos git:(master) ✗ grep -Ri acknowledgments src | wc -l
       6

Ditto for the other cases.

Reply:
Ok, I'll back out these adjustments.